src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.animal.AnimalDao;
+import core.basesyntax.dao.animal.AnimalDaoImpl;
+import core.basesyntax.dao.figure.FigureDao;
+import core.basesyntax.dao.figure.FigureDaoImpl;
+import core.basesyntax.dao.ma.CoachDao;
+import core.basesyntax.dao.ma.CoachDaoImpl;
+import core.basesyntax.dao.ma.MentorDao;
+import core.basesyntax.dao.ma.MentorDaoImpl;
+import core.basesyntax.dao.ma.PersonDao;
+import core.basesyntax.dao.ma.PersonDaoImpl;
+import core.basesyntax.dao.machine.MachineDao;
+import core.basesyntax.dao.machine.MachineDaoImpl;
+import core.basesyntax.model.figure.Circle;
+import core.basesyntax.model.figure.Triangle;
+import core.basesyntax.model.ma.Coach;
+import core.basesyntax.model.ma.Mentor;
+import core.basesyntax.model.ma.Person;
+import core.basesyntax.model.machine.Car;
+import core.basesyntax.model.machine.Machine;
+import core.basesyntax.model.machine.Truck;
+import core.basesyntax.model.zoo.Animal;
+import core.basesyntax.model.zoo.Cat;
+import core.basesyntax.model.zoo.Dog;
+import core.basesyntax.util.HibernateUtil;
+
+public class Main {
```

Comment:
I think we don't need to push Main to our pull request in this homework

Reply:
that's okay here,You  may leave it